### PR TITLE
Add MiW episode with transcript

### PR DESCRIPTION
**Pull request by zchase**
This PR adds the MiW episode with transcription to our webinar listing page. This PR also fixes the copy on webinar single pages to be in alignment with the current content of the Getting Started Guide (AWS static website).

### CODE REVIEW

File: content/webinars/aws-anomaly-detection-guardduty-slack/index.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    image: "/icons/containers.svg"
+
+# Content for the left hand side section of the page.
+main:
+    # Webinar title.
+    title: "Anomaly Detection with AWS GuardDuty and Slack"
+    # URL for embedding a URL for ungated webinars.
+    youtube_url: "https://www.youtube.com/embed/PF7KnOB3fLk"
+    # Sortable date. The datetime Hugo will use to sort the webinars in date order.
+    sortable_date: 2020-04-01T10:00:00-07:00
```
Comment (justinvp) [inline content/webinars/aws-anomaly-detection-guardduty-slack/index.md:53]: Is this right?

**Review by justinvp — APPROVED** [review]
LGTM.

One unrelated question: How does one actually navigate to `/webinars` from the homepage? I can't find any links to it anywhere so end up manually navigating to `https://www.pulumi.com/webinars`. Should we make this content more discoverable?

File: content/webinars/aws-anomaly-detection-guardduty-slack/index.md (hunk 1 above)
Comment (zchase) [reply content/webinars/aws-anomaly-detection-guardduty-slack/index.md:53]: Yup, that is the date the video was uploaded to YouTube. This is not a new video but rather I transcribed an old video for SEO purposes.

**Comment by zchase** [comment]
@justinvp We have an outstanding issue to combine the webinar and event pages and when we do that we will add navigation to the top nav. I have an in-progress PR here: https://github.com/pulumi/docs/pull/4172

**Pull request merged**